server/src/users/users.controller.ts

```diff
@@ -23,6 +24,11 @@ export class UsersController {
     return this.usersService.create(createUserDto);
   }
 
+  @Get()
```

Comment:
This route must be available only for the admin as mentioned in the issue description, So assign the admin role for this route.